src/waveform_config.hpp.in

```diff
@@ -3,8 +3,8 @@
 #cmakedefine DECORATE_SIMD_FUNCS
 
 #ifdef DECORATE_SIMD_FUNCS
-#define DECORATE_AVX2 __attribute__ ((__target__ ("avx2", "fma")))
-#define DECORATE_AVX __attribute__ ((__target__ ("avx", "fma")))
+#define DECORATE_AVX2 __attribute__ ((__target__ ("avx2,fma")))
```

Comment:
I double checked with

https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html

And it says:

> Providing multiple strings as arguments separated by commas to specify multiple options is equivalent to separating the option suffixes with a comma (‘,’) within a single string. Spaces are not permitted within the strings. 

So I think it should not break GCC compatibility.